CMake/ITKModuleExternal.cmake

```diff
@@ -50,6 +50,10 @@ if(NOT ITK_BINARY_DIR)
   set(ITK_BINARY_DIR ${ITK_DIR})
 endif()
 
+if(NOT CMAKE_INSTALL_LIBDIR)
+ set(CMAKE_INSTALL_LIBDIR lib)
```

Comment:
sometimes on 64 bit computers, this needs to be lib64.  

I think that ITK does not respect the OS requested lib name in all cases, and this solution works sometimes because most of ITK uses "lib" instead of the OS recommended library path name.

Reply:
I might be wrong but I think this commit mimics your [commit](https://github.com/InsightSoftwareConsortium/ITK/commit/5b09d546666f6110250aa7edbb59d4469791b944#) for external modules, doesn't it?